keps/sig-instrumentation/distributed-tracing-kep.md

```diff
@@ -122,11 +151,14 @@ This documentation should put forward standards for:
 Having these standards in place will ensure that our tracing instrumentation works well with all backends, and that reviewers have concrete criteria to cross-check PRs against. 
 
 
-### Risks and Mitigations
+### Graduation requirements
```

Comment:
Other things that should be included:

* User-facing documentation
* Alpha implementation as described above